src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -422,7 +498,7 @@ def init_results(self, redis_job_id: str, area: "AreaBase",
             self._export = CoefficientExportAndPlot(
                 area, self.export_path, self.export_subdir, self._endpoint_buffer)
 
-    def update_results(
+    def update_and_send_results(
```

Comment:
@spyrostz I found this thanks to pylint. But I think we should just overwrite the `update_and_send_results` in the `CoefficientSimulation` in order to also handle the pause functionality.
Otherwise the following error will be shown:
```
Traceback (most recent call last):
  File "/Users/hannesd/Envs/gsy-e/lib/python3.8/site-packages/rq/worker.py", line 1043, in perform_job
    rv = job.perform()
  File "/Users/hannesd/Envs/gsy-e/lib/python3.8/site-packages/rq/job.py", line 814, in perform
    self._result = self._execute()
  File "/Users/hannesd/Envs/gsy-e/lib/python3.8/site-packages/rq/job.py", line 837, in _execute
    result = self.func(*self.args, **self.kwargs)
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/exchange_jobs.py", line 40, in start
    launch_simulation_from_rq_job(**payload, job_id=current_job.id)
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/rq_job_handler.py", line 124, in launch_simulation_from_rq_job
    run_simulation(setup_module_name=scenario_name,
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/simulation.py", line 1054, in run_simulation
    simulation.run()
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/simulation.py", line 685, in run
    self._execute_simulation(initial_slot, tick_resume)
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/simulation.py", line 955, in _execute_simulation
    self._handle_paused(console)
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/simulation.py", line 831, in _handle_paused
    self._results.update_and_send_results(
  File "/Users/hannesd/gsy/gsy-e/src/gsy_e/gsy_e_core/simulation.py", line 504, in update_and_send_results
    raise NotImplementedError
```
Or did I misunderstand the reason for raising the `NotImplementedError` ?

Reply:
Ah I see now why overwriting is not possible: Because the interface is different.
The I need to handle it in the `_handle_paused` method and decide there which `SimulationResultsManager` is used

Reply:
Please have a look at my latest commit: https://github.com/gridsingularity/gsy-e/pull/1532/commits/bacf0e257a56dfee659bf6f64f44f819ccab9338

Reply:
Hmmm this will definitely work but I am thinking whether it makes sense to harmonize the interface if it is possible. I would recommend a separate PR though in order to be dealt with in isolation if possible. Or a TODO comment, even though it is something outside our usual workflow it can be added so that we notice it later along the way.

Reply:
I have an idea: One could add the scm manager instance to the `CoefficientSimulationResultsManager` via an extra step (method) and not pass it in the `update_send_coefficient_results`. This is easily done. If you agree, i can quickly add it.

Reply:
Interesting, therefore in order to update there will need to be 2 distinct method calls: one that updates the scm_manager, and one that actually calls the update_send_results (which would be overridden in the CoefficientSimulationResultsManager class, and would replace the update_send_coefficient_results). I like this and support this, thanks a lot for dealing with it !